src/components/Calendar/CalenderComponent/Day.tsx

```diff
@@ -0,0 +1,63 @@
+import React,{useContext, useEffect, useState} from 'react'
+import dayjs from 'dayjs';
+import GlobalContext from '../context/GlobalContext';
+
+
+let colours = "bg-green-300 bg-red-300 bg-indigo-300 bg-gray-300 bg-blue-300 bg-purple-300 hover:bg-green-400 hover:bg-red-400 hover:bg-indigo-400 hover:bg-gray-400 hover:bg-blue-400 hover:bg-purple-400" 
+
+
+
+export default function Day({day,rowIdx}:{day:any,rowIdx:any}) {
+  const [dayEvents,setDayEvents] = useState([])
+  const{ setShowEventModal , setDaySelected , filteredEvents , setSelectedEvent ,monthIndex }=
+  useContext(GlobalContext)
+
+  useEffect(() => {
+    const events = filteredEvents.filter((evt) => dayjs(evt.day).format("DD-MM-YY")  === day.format("DD-MM-YY"));
```

Comment:
define type of `evt` here. it is reading type `never`

Reply:
attaching error logs for reference

```
Type error: Property 'day' does not exist on type 'never'.
  14 |
  15 |   useEffect(() => {
> 16 |     const events = filteredEvents.filter((evt) => dayjs(evt.day).format("DD-MM-YY")  === day.format("DD-MM-YY"));
     |                                                             ^
  17 |     setDayEvents(events)
  18 |   },[filteredEvents,day]);
  19 |
```

Reply:
I have fixed it along with some other Type errors .